src/Illuminate/Validation/ValidationRuleParser.php

```diff
@@ -96,17 +95,11 @@ protected function explodeExplicitRule($rule, $attribute)
             return Arr::wrap($this->prepareRule($rule, $attribute));
         }
 
-        $rules = [];
-
-        foreach ($rule as $value) {
-            if ($value instanceof Date) {
-                $rules = array_merge($rules, explode('|', (string) $value));
-            } else {
-                $rules[] = $this->prepareRule($value, $attribute);
-            }
-        }
-
-        return $rules;
+        return array_map(
+            [$this, 'prepareRule'],
```

Comment:
You could use first-class callable syntax here:
```suggestion
            $this->prepareRule(...),
```